src/sentry/notifications/notifications/base.py

```diff
@@ -83,7 +83,7 @@ def template_path(self) -> str:
          - src/sentry/templates/path/to/example.txt
         then set `template_path` for the notification to `path/to/example`.
         """
-        pass
+        return "sentry/emails/default"
```

Comment:
Rather than changing this here should we use the default over here if we don't recognise the issue category? https://github.com/getsentry/sentry/blob/2b934d2dbcdc5b083fd9b86d0095e78475463a2e/src/sentry/notifications/notifications/rules.py#L53

I guess ideally we'd default to this if no template was defined for a given issue category. Is there some way we can check that?

Reply:
Hmm I was assuming new issue types would define a category, would that not be the case? We could do the default if nothing is defined too.